src/ToDo.UI/ToDo.UI.shproj

```diff
@@ -11,4 +11,4 @@
   <Import Project="ToDo.UI.projitems" Label="Shared" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.CSharp.targets" />
 
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Why this change? Can be removed from your PR no?

Reply:
I don't know, I forgot discard this changes